content/manager/upload-blueprint.md

```diff
@@ -6,43 +6,42 @@ draft: false
 weight: 400
 ---
 
-Before Cloudify Manager can deploy your blueprint, you must upload it. You can upload a blueprint using the CLI or (for Premium users) the Cloudify Web interface.
+Before you can deploy a blueprint, you must upload the blueprint to the Cloudify Manager. You can upload a blueprint using the CLI. Premium users can also upload using the Cloudiy Web Interface.
 
 Either use a blueprint that you have written or download an [example blueprint](https://github.com/cloudify-cosmo/cloudify-nodecellar-example) to upload.
 
-## Uploading via the CLI
 
-From the Cloudify command-line interface, you can upload your blueprint to Cloudify Manager. You must specify a path to a blueprint file. Cloudify compresses the folder and its contents.
+## Uploading a Blueprint using the Cloudify Web Interface
```

Comment:
Don't we call it Console now?

Reply:
I think we said that the new terms will be added in the first sprint of 4.4.

Reply:
OK :)